rs/src/management/http_client.rs

```diff
@@ -12,6 +12,9 @@ use url::Url;
 
 use rand::Rng;
 
+use winreg::enums::*;
+use winreg::RegKey;
```

Comment:
These have to be in conditional compilation as well, or it'll fail on other platforms. You can put them directly in the block down below.